testnet-contracts/contracts/CosmosBridge.sol

```diff
@@ -0,0 +1,295 @@
+pragma solidity ^0.5.0;
+
+import "../../node_modules/openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "./Valset.sol";
+import "./BridgeBank/BridgeBank.sol";
+
+contract CosmosBridge {
+
+    using SafeMath for uint256;
+
+    /*
+    * @dev: Public variable declarations
+    */
+    address public operator;
+    Valset public valset;
+    address public oracle;
+    bool public hasOracle;
+    BridgeBank public bridgeBank;
+    bool public hasBridgeBank;
+
+    uint256 public bridgeClaimCount;
+    mapping(uint256 => BridgeClaim) public bridgeClaims;
+
+    enum Status {
+        Null,
+        Pending,
+        Success,
+        Failed
+    }
+
+    struct BridgeClaim {
+        uint256 nonce;
+        bytes cosmosSender;
+        address payable ethereumReceiver;
+        address originalValidator;
+        address tokenAddress;
+        string symbol;
+        uint256 amount;
+        Status status;
+    }
+
+    /*
+    * @dev: Event declarations
+    */
+    event LogOracleSet(
+        address _oracle
+    );
+
+    event LogBridgeBankSet(
+        address _bridgeBank
+    );
+
+    event LogNewBridgeClaim(
+        uint256 _bridgeClaimCount,
+        uint256 _nonce,
+        bytes _cosmosSender,
+        address payable _ethereumReceiver,
+        address _validatorAddress,
+        address _tokenAddress,
+        string _symbol,
+        uint256 _amount
+    );
+
+    event LogBridgeClaimCompleted(
+        uint256 _bridgeClaimID
```

Comment:
I think bridgeClaimID should actually be bridgeProphecyID, right? implementation looks fine, i'm just refering to naming...and similarly, related functions should be like processProphecy instead of processClaim?

Reply:
https://github.com/cosmos/peggy/pull/82/commits/60cb52d4b6baf69bc7e237ba9839b1a9e2fae1a2